libs/types/src/cdf/ballot-definition/convert.ts

```diff
@@ -532,7 +521,7 @@ export function convertVxfElectionToCdfBallotDefinition(
     // we were to use the current date, it would cause changes every time we
     // hash the object. We want hashes to be based on the content of the
     // election, not the date generated.
-    GeneratedDate: dateTimeString(new Date(vxfElection.date)),
+    GeneratedDate: `${vxfElection.date.toISOString()}T00:00:00Z`,
```

Comment:
Is there a reason you're not using the `toMidnight` helper here, as you do below?

Reply:
Yeah, in this case I want it to be the same no matter what the system timezone is so it's stable for tests. This date doesn't actually matter for anything, it's just a required field in CDF.